js/app/src/components/StatusTracker/ToastContent.svelte

```diff
@@ -0,0 +1,275 @@
+<script lang="ts">
+	import { Alert, Information, Warning } from "@gradio/icons";
+	import { createEventDispatcher, onMount, afterUpdate } from "svelte";
+	import { fade } from "svelte/transition";
+	import type { ToastMessage } from "./types";
+
+	export let message: string = "";
+	export let type: ToastMessage["type"];
+	export let id: number;
+
+	const dispatch = createEventDispatcher();
+
+	function close_message() {
+		dispatch("close", id);
+	}
+
+	onMount(() => {
+		setTimeout(() => {
+			close_message();
+		}, 10000);
```

Comment:
Maybe we should make this configurable via a prop (also makes for easier testing).